idd/Energy+.idd.in

```diff
@@ -750,6 +750,22 @@ ZoneCapacitanceMultiplier:ResearchSpecial,
        \minimum> 0.0
        \note Used to alter the capacitance of zone air with respect to zone air generic contaminant concentration
 
+HVACSystemRootFindingAlgorithm,
+	   \memo Specifies a HVAC system solver algorithm to find a root
```

Comment:
@Myoldmopar I will clean up, since I am gong to make changes based your another comments.